### PR TITLE
Bug 1125026 - [NFC] Shrinking UI is blank when try to share received contact or file which opened from notification

### PR DESCRIPTION
**Pull request by cctuan**
https://bugzilla.mozilla.org/show_bug.cgi?id=1125026

### CODE REVIEW

File: apps/system/js/app_window.js [hunk 1]
```diff
@@ -1234,6 +1234,10 @@
    */
   AppWindow.prototype._showScreenshotOverlay =
     function aw__showScreenshotOverlay() {
+      if (this.frontWindow) {
```
Comment (alivedise) [inline apps/system/js/app_window.js]: this.frontWindow && this.frontWindow.isActive()

File: apps/system/js/app_window.js [hunk 2]
```diff
@@ -1256,6 +1260,9 @@
    */
   AppWindow.prototype._hideScreenshotOverlay =
     function aw__hideScreenshotOverlay() {
+      if (this.frontWindow) {
```
Comment (alivedise) [inline apps/system/js/app_window.js]: this.frontWindow && this.frontWindow.isActive()

**Comment by try-server-hook** [comment]
[cctuan George (cctuan)](https://github.com/cctuan) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=5b5f31061cb9)

**Comment by try-server-hook** [comment]
[cctuan George (cctuan)](https://github.com/cctuan) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=07d40c8b2f27)

**Comment by try-server-hook** [comment]
[cctuan George (cctuan)](https://github.com/cctuan) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=28187bae9479)

**Pull request merged**